packages/web-components/fast-foundation/src/disclosure/disclosure.ts

```diff
@@ -0,0 +1,100 @@
+import { attr, FASTElement } from "@microsoft/fast-element";
+/**
+ * A Disclosure Custom HTML Element.
+ * Based largely on the {@link https://w3c.github.io/aria-practices/#disclosure | disclosure element }.
+ *
+ * @public
+ */
+export class Disclosure extends FASTElement {
+    /**
+     * See {@link https://www.w3.org/WAI/PF/aria/roles#button} for more information
+     * @public
+     * @remarks
+     * HTML Attribute: aria-expanded
+     */
+    @attr({ attribute: "aria-expanded" })
+    public ariaExpanded: true | false;
+
+    /**
+     * Determines if the element should show the extra content or not.
+     *
+     * @public
+     */
+    @attr({ mode: "boolean" })
+    public expanded: boolean;
+
+    /**
+     * Invoker title
+     *
+     * @public
+     */
+    @attr
+    public title: string;
+
+    /**
+     * @internal
+     */
+    public details: HTMLDetailsElement;
+
+    /**
+     * @internal
+     */
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.setup();
+    }
+
+    /**
+     * @internal
+     */
+    public disconnectedCallback(): void {
+        super.disconnectedCallback();
+        this.details.removeEventListener("toggle", this.onToggle);
+    }
+
+    constructor() {
+        super();
+        this.ariaExpanded = false;
```

Comment:
This can likely be removed. If we want to default expanded, we can do that in connectedCallback():
```
if (!this.expanded) {
    this.expanded = false;
}
```